blocks/to-do/list-items.js

```diff
@@ -0,0 +1,8 @@
+export default async function getListItems() {
+  const response = await fetch('/to-do-form.json');
+  if (!response.ok) {
+    throw new Error('Response error');
```

Comment:
If you see `Response error` in the console, would you know immediately what failed? Better to be more explicit than less so. You can mention the requested file name, the error code/message or even the source method where it occurred